frontend/src/pages/accounts/settings.page.tsx

```diff
@@ -157,6 +164,43 @@ const Settings: NextPage = () => {
                     {labelCollectionWarning}
                   </div>
                 </div>
+                <div className={styles.field}>
+                  <h2 className={styles["field-heading"]}>
+                    {l10n.getString("setting-label-api-key")}
+                  </h2>
+                  <div
+                    className={`${styles["copy-api-key-input"]} ${styles["field-content"]}`}
+                  >
+                    {/* TODO: Convert the following element to <output> when "hidden" */}
+                    <div className={styles["copy-api-key-display"]}>
+                      {profile.api_token}
+                    </div>
+                    <span className={styles["copy-controls"]}>
+                      <span className={styles["copy-button-wrapper"]}>
+                        <button
+                          className={styles["copy-button"]}
+                          title={l10n.getString("settings-button-copy")}
+                          aria-label={l10n.getString("settings-button-copy")}
+                          onClick={copyApiKeyToClipboard}
+                        >
+                          <img
+                            src={copyIcon.src}
+                            alt=""
+                            className={styles["copy-icon"]}
+                          />
```

Comment:
suggestion (non-blocking): We should come up with a uniform way (probably what we will talk about @ work week) to use icons uniformly. I don't know if this is a nebula icon, but it looks like an icon and I know we have an `icons.tsx` file. Just to make things more predictable in the long run.

Reply:
We used to keep them all in `/static/` to ensure that the non-React and React website would use the same website. Now that we've converted to React, I think it makes sense to move them into `Icons.tsx` so that we can recolour icons as necessary - https://github.com/mozilla/fx-private-relay/pull/1891 actually does that, but of course, while it's awaiting a review, new PRs will arrive that don't yet have a consistent method to emulate.